tests/unit/graph_reader/test_artifact_collection.py

```diff
@@ -53,25 +55,19 @@
         ),
     ],
 )
-def test_one_session(execute, input_script, artifact_list):
+def test_one_session(linea_db, execute, input_script, artifact_list):
     """
     Test code refactor, make sure each artifact has a dedicated function in the
     generated module.
     """
 
     artifact_string = ", ".join([f'"{x}"' for x in artifact_list])
-    code = (
-        pathlib.Path(
-            "tests/unit/graph_reader/inputs/" + input_script
-        ).read_text()
-        + "from lineapy.graph_reader.artifact_collection import ArtifactCollection\n"
-        + "from lineapy.execution.context import get_context\n"
-        + "db = get_context().executor.db\n"
-        + f"ac = ArtifactCollection(db, [{artifact_string}])"
-    )
+    code = pathlib.Path(
+        "tests/unit/graph_reader/inputs/" + input_script
+    ).read_text()
 
-    res = execute(code, snapshot=False)
-    ac = res.values["ac"]
+    execute(code, snapshot=False)
+    ac = ArtifactCollection(linea_db, artifact_list)
```

Comment:
@lionsardesai: This change addresses LIN-561 (Declutter tests related to `ArtifactCollection`) as we discussed. Not sure if it results in speed improvement, but it's definitely less verbose code.